shortcodes.php

```diff
@@ -1027,6 +1028,40 @@ function sc_single_post($atts = Array())
 }
 add_shortcode('single_post', 'sc_single_post');
 
+/**
+ * Returns author bio markup for both
+ * single and featured posts
+ *
+ * @return string
+ * @author Cadie Brown
+ **/
+function sc_author_bio() {
```

Comment:
In general, it's a little dangerous to use the global `$post` object on a shortcode, since they can be put anywhere (including in "the loop"). It may be a little safer to:

1. Allow the post_id (or alternatively the author or author_id) to be passed in as a attribute.
2. Double check the `$post` object before falling back to it.

I think this will make the shortcode a little more usable throughout the site.